modules/snmp/snmp.go

```diff
@@ -95,23 +93,10 @@ func (s *SNMP) Init() bool {
 		return false
 	}
 
-	if len(s.ChartInput) > 0 {
-		s.charts, err = allCharts(s.ChartInput)
-		if err != nil {
-			s.Errorf("Population of charts failed: %v", err)
-			return false
-		}
-	} else {
-		c := defaultSNMPchart.Copy()
-		c.ID = fmt.Sprintf(c.ID, 1)
-		c.Title = fmt.Sprint(c.Title, "default")
-		for _, d := range defaultDims {
-			if err = c.AddDim(d); err != nil {
-				s.Errorf("Population of charts failed: %v", err)
-				return false
-			}
-		}
-		s.charts = &module.Charts{c}
+	s.charts, err = allCharts(s.ChartInput)
```

Comment:
Let's rename allCharts to newCharts

Reply:
we already have a function name `newChart()`. having another functions `newCharts()` at a glance is pretty confusing. `allCharts()` isn't a bad name afterall for this situation. Let's keep it this way

Reply:
But you can keep it as it is if you want, no problem.